modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -243,6 +243,20 @@ Example output plugin configuration:
     Tls          On
 ```
 
+==== Configure Log Forwarding
```

Comment:
Should we say  `Configure Log Forwarding From` ?
As this section is to enable/disable what logs needs to be forwarded.  @sundippatel